src/View/View.php

```diff
@@ -1255,11 +1255,14 @@ protected function _paths($plugin = null, $cached = true)
             }
         }
 
+        $class = new \ReflectionClass('\Cake\View\View');
+        $cake = dirname(dirname($class->getFilename()));
```

Comment:
Shouldn't `__DIR__` work then too, ie, is there actually an issue with the original code, I took the problem as a fact and haven't checked it?

AFAIR `__DIR__` and `__FILE__` do behave the same, and giving it a quick shot shows that overriding `View::_paths()` doesn't cause any problems, ie `$paths` always looks like

```php
[
	(int) 0 => '/www/htdocs/src/Template/',
	(int) 1 => '/www/htdocs/vendor/cakephp/cakephp/src/Template/'
]
```